src/Symfony/Component/Config/Definition/NodeInterface.php

```diff
@@ -83,6 +89,9 @@ public function merge($leftSide, $rightSide);
      * @param mixed $value The value to finalize
      *
      * @return mixed The finalized value
+     *
+     * @throws Exception
```

Comment:
All `@throws` should have a description explaining when the exception can be triggered... or be removed :)